CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Change Log
 
+## [13.0.0] - 2025-01-23
+
+### Added
+
+- automatically interpolate $TASK_WORKDIR in environment (#630)
```

Comment:
I didn't realize the title of that PR didn't change but this should be `{task_workdir}`